### PR TITLE
ETU-315 - fix(rxDataTable): adding tier, and visualizing score calculation

### PR DESCRIPTION
**Pull request by JoseVillarreal**
[ETU-315](https://jira.rax.io/browse/ETU-315)

### CODE REVIEW

File: dist/scripts/rx-data-table.js [hunk 1]
```diff
@@ -406,16 +406,29 @@ app.directive('rxDataTable', function ($http, $timeout, $document, $filter, $par
...
             });
 
-            scope.scoreTooltipTemplate = function (score) {
+            scope.scoreTooltipTemplate = function (row) {
                 var template = [
-                    'Waiting on Racker for ', $filter('rxAge')(score.lastResponseTime, 3, true)
+                    'Waiting on Racker for ', $filter('rxAge')(row.score.lastResponseTime, 3, true),
+                    'Score: ',
+                    '-(',
+                    row.score.weight,
```
Comment (gairehari) [inline dist/scripts/rx-data-table.js]: @JoseVillarreal: Do they just want to see the numbers or do they also need to know what those numbers mean ?

Comment (JoseVillarreal) [inline dist/scripts/rx-data-table.js]: Apparently just the numbers

**Comment by gairehari** [comment]
@JoseVillarreal: we need to fix this conflict

**Comment by gairehari** [comment]
LGTM

**Comment by gairehari** [comment]
No need to merge this PR, we are going to use the hash of last commit in bower in encote-tq-ui

**Comment by gairehari** [comment]
Seems the calculation has been updated in TS:
- (((maxResponseTime * tierWeight) - elapsedResponseTimeInSeconds) / threshold)

File: dist/scripts/rx-data-table.js [hunk 2]
```diff
@@ -408,16 +408,27 @@ app.directive('rxDataTable', function ($http, $timeout, $document, $filter, $par
 
             scope.scoreTooltipTemplate = function (row) {
                 var template = [
-                    'Waiting on Racker for ', $filter('rxAge')(row.score.lastResponseTime, 3, true)
+                    'Waiting on Racker for ', $filter('rxAge')(row.score.lastResponseTime, 3, true),
+                    'Score: ',
+                    '- (((',
+                    row.score.maxResponseTime,
+                    ' * ',
```
Comment (gairehari) [inline dist/scripts/rx-data-table.js]: @rnreekez Need to update the algorithm here

File: dist/scripts/rx-data-table.js [hunk 3]
```diff
@@ -408,16 +408,27 @@ app.directive('rxDataTable', function ($http, $timeout, $document, $filter, $par
...
                 return template.join(' ');
             };
 
             scope.maxResponseCalculation = function (row) {
                 var template = [
                     row.account_tier || '', //jshint ignore:line
-                    row.score.type, 'for ticket is ',
-                    $filter('rxAge')(moment().subtract(row.score.maxResponseTime, 'seconds'), 2, true)
+                    row.score.type,
+                    'for ticket is ', $filter('rxAge')(moment().subtract(row.score.maxResponseTime, 'seconds'), 2, true)
```
Comment (keithcruz) [inline dist/scripts/rx-data-table.js]: Any reason these two array elements need to be on the same line while the others don't?

**Comment by rnreekez** [comment]
@keithcruz @gairehari updates made and new dist files checked in.

**Comment by gairehari** [comment]
LGTM

**Comment by keithcruz** [comment]
LGTM

**Comment by samatha-kankipati** [comment]
LGTM

**Comment by gairehari** [comment]
LGTM

File: src/scripts/htmlPopover.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    .directive( 'popoverHtmlUnsafePopup', function () {
+        return {
+            restrict: 'EA',
+            replace: true,
+            scope: { title: '@', content: '@', placement: '@', animation: '&', isOpen: '&' },
+            templateUrl: 'src/templates/htmlPopover.html'
+        };
+    })
+    .directive( 'popoverHtmlUnsafe', [ '$tooltip', function ( $tooltip ) {
+        return $tooltip('popoverHtmlUnsafe', 'popover', 'click' );
```
Comment (bry4n) [inline src/scripts/htmlPopover.js]: spaces 😮

**Comment by bry4n** [comment]
added a comment about spacing but it's nitpick. the rest looks LGTM.

**Comment by bry4n** [comment]
LGTM

**Comment by keithcruz** [comment]
👍

**Pull request merged**